app/views/play/CampaignView.coffee

```diff
@@ -119,6 +119,8 @@ module.exports = class CampaignView extends RootView
               application.router.navigate((if me.isStudent() then '/students' else '/teachers'), {trigger: true, replace: true})
               noty({text: 'Please create or join a classroom first', layout: 'topCenter', timeout: 8000, type: 'success'})
           return
+      if @terrain is 'game-dev-hoc'
+        window.tracker?.trackEvent 'Started Campaign', category: 'Amazon', label: 'game-dev-hoc'
```

Comment:
This should be for all campaigns, and instead called `"Start HoC Campaign"` to be in present tense and more specifically for when links include the `hour_of_code` query param. This is also not necessarily an Amazon category. I'm kind of thinking maybe we should just not have an Amazon category?